app/views/layouts/application.html.erb

```diff
@@ -16,5 +16,6 @@
     <p class="alert"><%= alert %></p>
     <%= render "devise/shared/flashes" %>
     <%= yield %>
+    <%= render "shared/footer" %>
   </body>
 </html>
```

Comment:
in a` <footer>` element please.

And this is raising a conflict with master, you'll need to merge master in your branch to solve all of those
Ask me on slack if you're unsure what to do